modules/apps/transfer/types/packet.go

```diff
@@ -34,6 +34,20 @@ func NewFungibleTokenPacketData(
 	}
 }
 
+func NewFungibleTokenPacketDataWithMetadata(
```

Comment:
I believe @colin-axner meant that this function should be removed and the above function should take in a `metadata` field

Reply:
fixed